README.md

```diff
@@ -89,15 +89,15 @@ presents the detailed design of the controller.
 This is a high-level overview of the main priorities for 2023, in expected order of release:
 
 - Job preemption to reclaim borrowed quota and to accommodate high priority jobs [#83](https://github.com/kubernetes-sigs/kueue/issues/83), this is planned for v0.3
+- Cooperative preemption support with checkpoint mechanism [#477](https://github.com/kubernetes-sigs/kueue/issues/477)
```

Comment:
```suggestion
- Cooperative preemption support for workloads that implement checkpointing [#477](https://github.com/kubernetes-sigs/kueue/issues/477)
```

we won't be providing the checkpoint mechanism.

Reply:
sg.